packages/devtools_app/macos/Runner-Release.entitlements

```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>com.apple.security.app-sandbox</key>
-	<true/>
+	<false/>
```

Comment:
We should add a TODO to restore this value to true before we consider releasing a desktop version of the app.

This is the sort of thing we could forget about and then become upset to discover was not sandboxed.

Reply:
I may be misunderstanding what this flag does but I think it is necessary for us to turn this off given we want to create websocket connections from this debugging tool.